psi4/src/psi4/libmints/basisset.h

```diff
@@ -281,6 +281,10 @@ class PSI_API BasisSet {
     /// Return the number of core electrons associated with this (ECP) basisset, for the specified label.
     int n_ecp_core(const std::string &label) const { return ncore_.count(label) ? ncore_.at(label) : 0; }
 
+    /// Helper functions for frozen core to reduce LOC
+    int _atom_to_period(int Z);
```

Comment:
These are internal functions for the class? If so, what's signaled in py with the underscore prefix should in c++ remove the underscore and move to the private: section of the header.